tfhe/src/lib.rs

```diff
@@ -71,7 +75,8 @@
 )]
 #![cfg_attr(all(doc, not(doctest)), feature(doc_auto_cfg))]
 #![cfg_attr(all(doc, not(doctest)), feature(doc_cfg))]
-#![warn(rustdoc::broken_intra_doc_links)]
+// Weird clippy lint triggering without any code location
+#![cfg_attr(test, allow(clippy::large_stack_arrays))]
```

Comment:
Could have a common comment with large_stack_frames I think

Reply:
could be,